src/middleware/security.ts

```
@@ -0,0 +1,251 @@
+
+// Bot rate limiting - Track user requests to prevent spam
+export const botRateLimit = createRateLimit(
+  60 * 1000, // 1 minute
+  10, // 10 requests per minute
+  'Too many bot requests, please slow down.',
+  (req) => req.body?.message?.from?.id?.toString() || req.ip
+);
+
+// Roll number validation
+export const rollNumberValidation = [
+  body('rollNumber')
```

ESLint

[error] Replace `'rollNumber'` with `"rollNumber"`

(prettier/prettier)